src/mapping-entry.ts

```diff
@@ -31,7 +31,8 @@ export function getAbsoluteMappingEntries(
   for (const key of sortedKeys) {
     absolutePaths.push({
       pattern: key,
-      paths: paths[key].map((pathToResolve) =>
+      paths: paths[key].map((pathToResolve) =>                    
+        if (path.isAbsolute(pathToResolve)) return pathToResolve;
         path.join(absoluteBaseUrl, pathToResolve)
```

Comment:
It looks like this is missing curly braces; you probably want this:
```ts
      paths: paths[key].map((pathToResolve) => {                 
        if (path.isAbsolute(pathToResolve)) return pathToResolve;
        return path.join(absoluteBaseUrl, pathToResolve);
     },

Reply:
fixed

Reply:
Wow quick fix, thanks! Hope this is merged soon.

Reply:
@jonaskello